shared/src/test/scala/mlscript/DiffTests.scala

```diff
@@ -174,7 +176,8 @@ class DiffTests
       expectCodeGenErrors: Bool = false,
       showRepl: Bool = false,
       allowEscape: Bool = false,
-      mono: Bool = false,
+      simpledef: Bool = false,
+      postProcessAfterTyping: Bool = false,
```

Comment:
I really don't like this. When we discussed this, we agreed to add a _new_ overridable hook after typing, not to make the semantics of the existing one depend on some obscure flag...